Bld/build.ps1

```diff
@@ -6,6 +6,13 @@ git submodule update --init --recursive
 
 # Run the build! :D
 dotnet build -c Release
+
+pushd Src/PRuntimes/RvmRuntime
+mvn install
+popd
+
+Copy-Item -Path "Src/PRuntimes/RvmRuntime/target/*.jar" -Destination "Bld/Drops"
```

Comment:
Can we please put it in `Bld/Drops/RVM`? for all platforms.

Reply:
I updated the both build scripts. But I only test the build.sh. I don't have a windows machine to test the build.ps1. please double check the script.

Reply:
We can check that the scripts work on windows when we add Github actions for running the RVM regressions.